zerver/lib/request.py

```diff
@@ -155,7 +155,7 @@ def _wrapped_view_func(request, *args, **kwargs):
                 try:
                     val = ujson.loads(val)
                 except Exception:
-                    raise JsonableError(_('argument "%s" is not valid json.') % (param.post_var_name,))
+                    raise JsonableError(_('Argument "%s" is not valid JSON.') % (param.post_var_name,))
```

Comment:
Make sure to change the unit test for this also. Seems to be the reason for failing backend tests.